### ISSUE TITLE
Step 16 - google maps  - 'GOOGLE_MAPS_PROVIDERS'

### ISSUE THREAD

**Issue by FaberNa** [Urigo/meteor-angular2.0-socially#150]
When i write in my editor or i start meteor project i have this problem : 

> node_modules/angular2-google-maps/core/index"' has no exported member 'GOOGLE_MAPS_PROVIDERS'

But for now angular2-google-maps it's installed. Also i have little problem of unmet dependency the i posted on stack overflow with this link 
http://stackoverflow.com/questions/39420064/meteor-rxjs5-0-0-beta-11-or-rxjs5-0-0-beta-6-unmet-dependency

**Comment by FaberNa** [comment]
Just i think this is the problem google maps version 0.14.0 require version rxjs 5.0.0-beta.11 instead angular @angular/router@3.0.0-alpha.8 requires a peer of rxjs@5.0.0-beta.6 , what is the solution ?? 
Install version of google map 0.12.0 ? but in package.json the notation says ^ and this mean that it's possible to install every package with major version 0, and if i don't have mistake , 0.14.0 have major version as 0.12.0.

**Comment by FaberNa** [comment]
Now i have update more component, and at least my package.json is : 

`{
  "name": "angular2-meteor-base",
  "private": true,
  "scripts": {
    "start": "meteor run",
    "test": "meteor test --driver-package practicalmeteor:mocha",
    "test:ci": "meteor test --once --driver-package dispatch:mocha-phantomjs"
  },
  "devDependencies": {
    "chai": "3.5.0",
    "chai-spies": "0.7.1"
  },
  "dependencies": {
    "@angular/common": "^2.0.0-rc.6",
    "@angular/compiler": "^2.0.0-rc.6",
    "@angular/core": "^2.0.0-rc.6",
    "@angular/forms": "^2.0.0-rc.6",
    "@angular/platform-browser": "^2.0.0-rc.6",
    "@angular/platform-browser-dynamic": "^2.0.0-rc.6",
    "@angular/router": "^3.0.0-rc.2",
    "@angular2-material/button": "^2.0.0-alpha.6",
    "@angular2-material/card": "^2.0.0-alpha.6",
    "@angular2-material/checkbox": "^2.0.0-alpha.6",
    "@angular2-material/core": "^2.0.0-alpha.6",
    "@angular2-material/input": "^2.0.0-alpha.6",
    "@angular2-material/toolbar": "^2.0.0-alpha.6",
    "angular2-google-maps": "^0.14.0",
    "angular2-meteor": "0.6.2",
    "angular2-meteor-accounts-ui": "^0.6.1",
    "angular2-meteor-auto-bootstrap": "0.6.0",
    "angular2-meteor-polyfills": "0.1.1",
    "angular2-meteor-tests-polyfills": "0.0.2",
    "es6-shim": "0.35.1",
    "meteor-node-stubs": "0.2.3",
    "ng2-pagination": "^0.3.5",
    "reflect-metadata": "0.1.3",
    "rxjs": "^5.0.0-beta.11",
    "zone.js": "^0.6.17"
  }
}`

**Comment by FaberNa** [comment]
At least my package.json is :
`

{
  "name": "angular2-meteor-base",
  "private": true,
  "scripts": {
    "start": "meteor run",
    "test": "meteor test --driver-package practicalmeteor:mocha",
    "test:ci": "meteor test --once --driver-package dispatch:mocha-phantomjs"
  },
  "devDependencies": {
    "chai": "3.5.0",
    "chai-spies": "0.7.1"
  },
  "dependencies": {
    "@angular/common": "^2.0.0-rc.6",
    "@angular/compiler": "^2.0.0-rc.6",
    "@angular/core": "^2.0.0-rc.6",
    "@angular/forms": "^2.0.0-rc.6",
    "@angular/platform-browser": "^2.0.0-rc.6",
    "@angular/platform-browser-dynamic": "^2.0.0-rc.6",
    "@angular/router": "^3.0.0-rc.2",
    "@angular2-material/button": "^2.0.0-alpha.6",
    "@angular2-material/card": "^2.0.0-alpha.6",
    "@angular2-material/checkbox": "^2.0.0-alpha.6",
    "@angular2-material/core": "^2.0.0-alpha.6",
    "@angular2-material/input": "^2.0.0-alpha.6",
    "@angular2-material/toolbar": "^2.0.0-alpha.6",
    "angular2-google-maps": "^0.14.0",
    "angular2-meteor": "0.6.2",
    "angular2-meteor-accounts-ui": "^0.6.1",
    "angular2-meteor-auto-bootstrap": "0.6.0",
    "angular2-meteor-polyfills": "0.1.1",
    "angular2-meteor-tests-polyfills": "0.0.2",
    "es6-shim": "0.35.1",
    "meteor-node-stubs": "0.2.3",
    "ng2-pagination": "^0.3.5",
    "reflect-metadata": "0.1.3",
    "rxjs": "^5.0.0-beta.11",
    "zone.js": "^0.6.17"
  }
}

`
No more problems with DEP, but for now the project doesn't run :(

**Comment by jasonburrows** [comment]
I have this problem as well... it is also strange in that even when I remove the '^' to try to lock it in at version 0.12.0, meteor just adds the '^' back in whenever I run an npm command.

I'm not sure how it ever worked as the project as checked in uses angular2 rc 4, but google maps 0.12 complains that it needs rc 2 and 0.13 complains that it needs rc 5.  Not sure what to do here.  

I also updated my angular version to try to just get on 2.0.0 since it was released, but the whole project dies even though it solves my dependancy problems.
